### PR TITLE
fix(locale): copies locales that are not translated externally

### LINKED ISSUE #91: Add a right_before_production_deploy_script to copy locales
This script will do all the copying and should run right before prod deploy to get the most translations.
This is a bit of opposite to the "extract_strings", that one runs right after prod is built.

- [ ] Automatically copy/symlink "pt-PT" to "pt" https://github.com/mozilla/fxa-content-server-l10n/issues/81
- [ ] We might need to copy `hi-IN` to `hi`
- [ ] `sv_SE` => `sv`
- [ ] Remove all existing copy task in content-server  https://github.com/mozilla/fxa-content-server/blob/master/grunttasks/copy.js#L86 and copy them as part of this script

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #91

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,17 +22,27 @@ brew link gettext --force # brew does not link the tools into /usr/local. Force
 
 Ubuntu: `sudo apt-get install gettext`
 
+The process to work with strings:
+
+* Make sure you installed dependencies and `msgfilter` (from `gettext`) command is available in your shell.
+* Have the fxa-auth-mailer and fxa-content-server repos at the directory level as fxa-content-server-l10n.
```
Comment (rfk) [inline README.md]: nit: "at the *same* directory level"?

File: scripts/before_prod_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+do
+param="$1"
+
+case $param in
+    --l10n-repo)
+    L10N_DIR="$2"
+    shift 2
+    ;;
+    *)
+    usage
```
Comment (rfk) [inline scripts/before_prod_deploy.sh]: `usage` is undefined?

**Comment by rfk** [comment]
The code change here looks pretty small and solid (modulo nits) so I'm happy to r+ this from a mechanical standpoint, if you and @shane-tomlinson and previously agreed on the details of the approach.

File: scripts/before_prod_deploy.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    ;;
+esac
+done
+
+# some locales must be copied
+# see details in https://github.com/mozilla/fxa-content-server-l10n/pull/79#issuecomment-149281761
+cp -r $L10N_DIR/locale/sv_SE/* $L10N_DIR/locale/sv
+cp -r $L10N_DIR/locale/hi_IN/* $L10N_DIR/locale/hi
+cp -r $L10N_DIR/locale/pt_PT/* $L10N_DIR/locale/pt
```
Comment (vladikoff) [inline scripts/before_prod_deploy.sh]: Add Serbian latin to this

**Comment by vladikoff** [comment]
Thanks Ryan!

**Pull request merged**